redfish-core/lib/network_protocol.hpp

```diff
@@ -238,30 +238,29 @@ inline void getNetworkData(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                          std::bind_front(afterNetworkPortRequest, asyncResp));
 } // namespace redfish
 
-inline void handleNTPProtocolEnabled(
-    const bool& ntpEnabled, const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
+inline void afterSetNTP(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                        const boost::system::error_code& ec)
 {
-    std::string timeSyncMethod;
-    if (ntpEnabled)
-    {
-        timeSyncMethod = "xyz.openbmc_project.Time.Synchronization.Method.NTP";
-    }
-    else
+    if (ec)
     {
-        timeSyncMethod =
-            "xyz.openbmc_project.Time.Synchronization.Method.Manual";
+        BMCWEB_LOG_DEBUG("Failed to set elapsed time. DBUS response error {}",
```

Comment:
Would this is `BMCWEB_LOG_ERROR()`?

Reply:
`BMCWEB_LOG_DEBUG() `would also be fine? - https://gerrit.openbmc.org/c/openbmc/bmcweb/+/69948/23/redfish-core/lib/network_protocol.hpp#246 ?

Reply:
@Pavithrab7 
I think - it needs to be updated to ERROR for 1110 and also for upstream, as it generates `internalError()`.

Reply:
updated to ERROR for 1110.